src/main/java/com/gamedoora/backend/userservices/assembler/UserServicesAssembler.java

```diff
@@ -75,13 +75,13 @@ public UserDTO createUsers(UserDTO userDto) {
     return userDto;
   }
 
-  public UserDTO updateUsers(long id, UserDTO userDto) {
+  public UserDTO updateUsers(String emailId, UserDTO userDto) {
 
-    Optional<Users> usersRes = usersRepository.findById(id);
-    if (usersRes.isPresent()) {
+    Users usersRes = usersRepository.findByEmail(emailId);
+    if (usersRes.getEmail().isEmpty()) {
```

Comment:
Can you please explain this check?

Reply:
This is to make sure that if a user with the given email Id does not exists, return null, just like we previously handle the checks, only difference being this time is that we are not using optional tag so we need to verify whether there exists any such user or not, if not then return null.

Reply:
This needs to be modified as isEmpty() will check length and not for null.